src/braket/pennylane_plugin/ahs_device.py

```diff
@@ -44,6 +44,15 @@
 from pennylane._version import __version__
 from pennylane.measurements import MeasurementProcess, SampleMeasurement
 from pennylane.ops import CompositeOp, Hamiltonian
+
+try:
+    from pennylane.ops import LinearCombination
+except (AttributeError, ImportError):
+
+    class LinearCombination:
+        pass
```

Comment:
Rather than doing conditional imports, I'd say it's fine to wait until 0.36 is released and import normally